scene/resources/shader.cpp

```diff
@@ -120,7 +120,7 @@ void Shader::get_shader_uniform_list(List<PropertyInfo> *p_params, bool p_get_gr
 			if (default_textures.has(pi.name)) { //do not show default textures
 				continue;
 			}
-			params_cache[pi.name] = pi.name;
+			params_cache[pi.name] = "shader_uniform/" + pi.name;
```

Comment:
What is the point of the map if the difference is just that `shader_uniform/` is prepended?
There is a comment saying `//map a shader param to a material param..` but it's not very descriptive of what that really means. I guess "shader param" is the name of the parameter as seen in the shader? And values are the name of the parameter as saved in the material `.tres` file? In which case, prepending `shader_uniform/` in `has_uniform` isn't necessary as the map could be looked up by key.

Reply:
@MGilleronFJ 
It seems that `shader_uniform/` is intended to be used to store the parameters to .tres file and RenderingServer. So, what this code doing is completely opposite from the expected behavior. 
Considering other compatibility issues, it would be better to fix this bug based on PR #64092.

Reply:
> and RenderingServer

That sounds wrong